src/chainlit/__init__.py

```diff
@@ -24,6 +24,7 @@
     Video,
 )
 from chainlit.haystack import HAYSTACK_INSTALLED
+from chainlit.input_widget import ChatSettings, SelectInput, Slider, Switch, TextInput
```

Comment:
as the project grow, I think we are going to stop putting everything in the main cl module. For new features I think it is ok to import them directly from the submodule